sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -112,6 +113,7 @@ def __init__(self, name, init_config, instances):
         self.databases = set()
         self.autodiscovery_query = None
         self.ad_last_check = 0
+        self.index_usage_last_check_ts = 0
```

Comment:
I recognize that `ad_last_check` is not an internal attribute, but both this and that one should be prefixed with `_`